tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/event_block.html

```diff
@@ -0,0 +1,54 @@
+{% load wagtailimages_tags util_tags %}
+
+<div class="grid__event event-block">
+    <div class="event-block__content">
+        <a class="event-block__title" href="{{ value.get_button_link }}" aria-label="{{ value.title }}">
+            <h2>
+                {{ value.title }}
+                {% include "patterns/atoms/icons/icon.html" with name="arrow-wide" classname="listing-avatar__arrow listing__arrow" %}
```

Comment:
Have a look at the listing styles and html here as we need to ensure this arrow doesn't wrap to the next line:

https://github.com/torchbox/torchbox.com/blob/integration/sprint-4/tbx/project_styleguide/templates/patterns/molecules/listing/listing--simple.html#L3-L6

https://github.com/torchbox/torchbox.com/blob/integration/sprint-4/tbx/static_src/sass/components/_listing.scss#L71-L79

Reply:
Fixed in 01ba3b7bf115fe2b9651aa60dc38f856bb83dbfc

Reply:
Added in 06f8cf118083ad8927e9644eacc89b9f3fd90b9f